app/notifiers/panopticon_notifier.rb

```diff
@@ -0,0 +1,14 @@
+require 'gds_api/panopticon'
+require 'ostruct'
+
+class PanopticonNotifier
+  def self.call
```

Comment:
Would it be better to make this an instance method? That makes the class [more refactorable](http://blog.codeclimate.com/blog/2012/11/14/why-ruby-class-methods-resist-refactoring/).